Potjans_2014/bm_helpers.py

```diff
@@ -2,17 +2,59 @@
 import os
 
 import nest
+import numpy as np
 
 
-def logging(py_timers=None, memory_used=None):
+def logging(py_timers=None, memory_used=None, intermediate_kernel_status=None):
```

Comment:
I suggest setting the default value of intermediate_kernel_status to an empty dict instead of None. Otherwise, for default parameters, the code will generate an error in the try and except statement.